src/src/Commands/CustomTests/RunE2ECommand.php

```diff
@@ -273,6 +273,7 @@ protected function execute( InputInterface $input, OutputInterface $output ): in
 
 		putenv( 'QIT_UP_AND_TEST=1' ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
 		putenv( sprintf( 'QIT_SUT=%s', $woo_extension ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
+		putenv( sprintf( 'QIT_SUT_SOURCE=%s', $source ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
```

Comment:
@MrJnrman what about we add realpath here?

Reply:
That'd do the trick